pkg/config/controller/config.go

```diff
@@ -69,6 +69,8 @@ type ControllerConfig struct {
 	IPsecCSRSignerConfig IPsecCSRSignerConfig `yaml:"ipsecCSRSigner"`
 	// Multicluster configuration options.
 	Multicluster MulticlusterConfig `yaml:"multicluster,omitempty"`
+	// AuthenticationOption provides different options for client auth when accessing antrea-controller's api-server.
+	Authentication AuthenticationOption `yaml:"authentication,omitempty"`
```

Comment:
Name it `APIServerAuthentication` to be clear?

Reply:
done.